### PR TITLE
Add whistle and new sound for emotion

### PR DESCRIPTION
**Pull request by Klaypexx**
## About the PR
Added whistle sound and sound for salute and crying (crying sound support only human and slime). 

**Media**

https://user-images.githubusercontent.com/68664576/235305463-67503f55-1c8e-4ee1-b9f8-3744e53b7284.mov

https://user-images.githubusercontent.com/68664576/235305706-d3f615ac-d0c6-444e-a856-fa8eac39227d.mov

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added whistling emotion
- add: Added sounds for the emotion of salute and crying

### CODE REVIEW

File: Resources/Audio/Voice/Human/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  copyright: "Taken from https://freesound.org/"
+  source: "https://freesound.org/people/Luzanne0/sounds/445299/"
+
+- files:
+  - cry_female_3.ogg
+  - cry_female_4.ogg
+  license: "CC-BY-4.0"
+  copyright: "Taken from https://freesound.org/"
+  source: "https://freesound.org/people/Idalize/sounds/408211/"
+
```
Comment (metalgearsloth) [inline Resources/Audio/Voice/Human/attributions.yml:47]: These licences aren't correct, e.g. one is sampling 1.0 and another is NC

File: Resources/Audio/Effects/Emotes/attributions.yml [hunk 2]
```diff
@@ -18,3 +18,9 @@
   license: "CC-BY-4.0"
   copyright: "Made by TV_LING. Converted from WAV to OGG and downmixed to 1 channel."
   source: "https://freesound.org/people/TV_LING/sounds/523467/"
+
+- files:
+    - salute.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Taken from https://github.com/ss220-space/Paradise"
+  source: "https://github.com/ss220-space/Paradise/tree/master220/sound/effects"
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/Emotes/attributions.yml:26]: Need to link commit not branch.

File: Resources/Audio/Effects/Emotes/sources.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+salute.ogg are taken from https://github.com/ss220-space/Paradise/tree/master220/sound/effects
```
Comment (metalgearsloth) [inline Resources/Audio/Effects/Emotes/sources.txt:1]: attributions.yml and as above.

File: Resources/Audio/Voice/Human/license.txt [hunk 4]
```diff
@@ -23,3 +23,17 @@ male_sneeze_1.ogg taken from https://freesound.org/people/InspectorJ/sounds/3521
...
+
+cry_male_1.ogg, cry_male_2.ogg are taken from https://freesound.org/people/jacobmathiassen/sounds/254869/
+
+cry_male_3.ogg taken from https://freesound.org/people/scottemoil/sounds/263776/
+
+cry_male_4.ogg taken from https://freesound.org/people/qubodup/sounds/200428/
+
+cry_female_1.ogg, cry_female_2.ogg are taken from https://freesound.org/people/Luzanne0/sounds/445299/
+
+cry_female_3.ogg, cry_female_4.ogg are taken from https://freesound.org/people/Idalize/sounds/408211/
```
Comment (metalgearsloth) [inline Resources/Audio/Voice/Human/license.txt:39]: attributions.yml

**Comment by Klaypexx** [comment]
*emotional damage*  🥲

**Comment by Klaypexx** [comment]
For some unknown reason there is still a conflict with the master and my branch. Probably it's because of the version dissolution, because there were no such problems on the updated branch I wanted to re-download

**Comment by Klaypexx** [comment]
Damn, I'm sorry, I made the merge in the wrong place

**Pull request closed without merging**